examples/plot_decoding_tutorial.py

```diff
@@ -222,11 +221,24 @@
 #
 # Let's leave out the 30 last data points during training, and test the
 # prediction on these 30 last points:
-svc.fit(fmri_masked[:-30], conditions[:-30])
+import numpy as np
+n_samples = len(conditions)
+fmri_niimgs_train = index_img(fmri_niimgs, np.arange(0, n_samples - 30))
```

Comment:
You could use a slice object here, instead of np.arange (code below not checked)
```suggestion
fmri_niimgs_train = index_img(fmri_niimgs, slice(0, -30))
```
As a result, you no longer need to define n_samples and to import numpy